addons/website_crm/views/website_crm_templates.xml

```diff
@@ -1,87 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<odoo>
-        <template id="contactus_form" name="Contact Form (Opportunity)" inherit_id="website.contactus_form" customize_show="True">
-            <xpath expr="//div[@id='contactus_section']" position="replace">
-                <div id="contactus_section">
-                    <section class="s_website_form" data-vcss="001" data-snippet="s_website_form">
-                        <div class="container">
-                            <form id="contactus_form" action="/website/form/" method="post" enctype="multipart/form-data" data-model_name="crm.lead" data-success-mode="redirect" data-success-page="/contactus-thank-you" class="o_mark_required" data-mark="*" data-editable-form="false">
```

Comment:
I am not too sure about that, does that mean when `website_crm` module is installed, there won't be any lead created when contact form is submitted :thinking: ?
That's the whole point of the module. Maybe I am missing something here :)

Reply:
@sbuhl ?